dluhc-component-library/src/components/siteSelectionForm/SiteSelectionForm.tsx

```diff
@@ -110,6 +115,15 @@ const SiteSelectionForm = ({ filepath, data, uiSchema }: SiteSelectionForm) => {
   const [activePage, setActivePage] = useState(0);
 
   useEffect(() => {
+    getFormData().then((retrievedFormData?: FormState) => {
+      setFormData(retrievedFormData ?? {});
+    });
+  });
```

Comment:
We have missed the second param of the useEffect Function.

The second param dictates when the useEffect should run by giving it an array of values that it watches. It only runs when any of the values change. In our case we only want it to run on the first loading of the component.

We should add an empty array like this `useEffect(() => ..., [])` which means it only runs once.